domain-model/src/test/java/md/maib/retail/model/RuleTest.java

```diff
@@ -52,7 +51,7 @@ void testRule() {
         // Rule Assertions
         assertThat(rule.getId()).isNotNull();
         assertThat(rule.getConditions()).containsExactly(condition);
-        assertThat(rule.getEffects()).containsExactly(effect);
+        Assertions.assertThat(rule.getEffects()).containsExactly(effect);
```

Comment:
Rule is a entity. It has state and can be changed. For equality of Rule we use ID, and EqualsVerifier will be preferred